File: src/main/java/ro/ubb/dp1819/exercises/entity/Espresso.java
package ro.ubb.dp1819.exercises.entity;

import java.util.Arrays;

public class Espresso extends Coffee{

    public Espresso(){
        ingredients = Arrays.asList(new Ingredient(2.0f, "cups", "water"),
                new Ingredient(0.25f, "cups", "coffee-beans", "roasted")
        );
    }

    @Override
    public String toString() {
        return "Expresso: "+ingredients.toString();
    }
}

File: anaspoiala-932-dotnet/Lab01_CreationalPatterns/Entities/Espresso.cs
using System.Collections.Generic;

namespace CoffeeMachine.Entities
{
	public class Espresso : Coffee
	{
		// 2 cups water 
		// 0.25 cups coffee-beans roasted
		private static new List<CoffeeIngredient> ingredients = new List<CoffeeIngredient>
		{
			new CoffeeIngredient(2f, "cups", "water"),
			new CoffeeIngredient(0.25f, "cups", "coffee-beans", "roasted")
		};

		public static new List<CoffeeIngredient> Ingredients { get => ingredients; }

		public override string Drink()
		{
			return $"Drinking an Espresso.";
		}
	}
}
